packages/stark-ui/src/modules/table/components/_table.component.scss

```diff
@@ -1,26 +1,21 @@
-//TODO: MatIcon size
-/*@mixin md-icon-size($size: 24px) {
-  font-size: $size;
-  height: $size;
-  width: $size;
-}
-
-.mat-icon{
-  @include md-icon-size(16px);
-  font-weight: inherit;
-  margin-left: 5px;
-}*/
-
 .stark-table {
   .header {
     align-items: center;
     display: flex;
     justify-content: space-between;
+    margin-bottom: 20px;
+
+    .transcluded{
+      flex-grow: 1;
+    }
 
     .actions {
       align-items: center;
       display: flex;
       justify-content: flex-end;
+      &.actions-alt{
```

Comment:
Could you please add a space before the bracket ?
(Prettier should do it automatically normally 😕)

Reply:
Strange cause I ran prettier-check before pushing... Maybe we should have a ticket about it?

Reply:
Well, in fact Prettier doesn't support HTML yet 😩
https://prettier.io/docs/en/language-support.html
https://github.com/prettier/prettier/issues/1882